docker-compose.yml

```diff
@@ -20,42 +20,42 @@ services:
     ports:
       - "8082:8081"
     environment:
-      DATABASE_URL: "${HASURA_GRAPHQL_DATABASE_URL}"
+      DATABASE_URL: "${HASURA_GRAPHQL_DATABASE_URL}?sslmode=disable"
     depends_on:
       - postgres
 
   hapi:
     container_name: eosrate_hapi
     build:
-      context: ./services/hapi
+      context: ./hapi
       dockerfile: Dockerfile
     volumes:
-      - ./services/hapi:/opt/application
+      - ./hapi:/opt/application
       - /opt/application/node_modules
     ports:
       - "9090:9090"
     environment:
       HAPI_SERVER_ADDRESS: "${HAPI_SERVER_ADDRESS}"
       HAPI_SERVER_PORT: "${HAPI_SERVER_PORT}"
-      POSTGRES_USER: "${POSTGRES_USER}"
-      POSTGRES_PASSWORD: "${POSTGRES_PASSWORD}"
-      POSTGRES_PORT: "${POSTGRES_PORT}"
-      POSTGRES_DB: "${POSTGRES_DB}"
-      POSTGRES_HOST: "${POSTGRES_HOST}"
-      POSTGRES_SCHEMA: "${POSTGRES_SCHEMA}"
-      EOS_API_ENDPOINT: "${EOS_API_ENDPOINT}"
+      HAPI_POSTGRES_USER: "${POSTGRES_USER}"
+      HAPI_POSTGRES_PASSWORD: "${POSTGRES_PASSWORD}"
+      HAPI_POSTGRES_PORT: "${POSTGRES_PORT}"
+      HAPI_POSTGRES_DB: "${POSTGRES_DB}"
+      HAPI_POSTGRES_HOST: "${HAPI_POSTGRES_HOST}"
```

Comment:
@xavier506  Wasn't it `POSTGRES_HOST`?

Reply:
@leisterfrancisco  we need this to be  `HAPI_POSTGRES_HOST`

`POSTGRES_HOST = localhost`
 `HAPI_POSTGRES_HOST = eosrate_postgres `

`localhost` only works inside the postgres pod, other services need to resolve a hostname such as `eosrate_postgres`